### DISCUSSION TITLE
Sign Language Support

### DISCUSSION THREAD

**Discussion by nathancoooper** [kantord/LibreLingo#2179] [Q&A] [answered]
As a language enthusiast (focusing on English, French, Latin, Japanese, and British Sign Language), I was wondering whether there was any support for animated GIFs ergo allowing for the creation of sign language courses? I am about to be on summer break from college in a month and can't wait to start building some new courses!

**Comment by kantord** [comment, marked as answer]
There's no current active support for sign language or animated GIFs. Perhaps you can make some wireframes/mockups of how this could work?

**Reply by nathancoooper** [reply]
That's alright, I'll note it down and look into it in a few months once I have more free time.

**Reply by kantord** [reply]
by the way, are there currently software for input using sign language? because for a sign language course, I think input could also be very important. Ideally it would be an input mechanism that is built into browsers, or something (open source) that we can embed into LibreLingo

**Reply by nathancoooper** [reply]
I believe the people over at  [Synapse](https://github.com/albanjoseph/Signapse/) are working on ways of using device cameras for this sort of kinesics recognition.

**Reply by kantord** [reply]
That's really really cool! And I think it should be possible to integrate it with LibreLingo somehow. Although, unless the users use a generic browser integration with it, it will probably be a bit complicated because it will require us to create a desktop app probably :thinking:
